changelog.json

```
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.1.2-beta1": [
+      "[Added] Hide whitespace in historical commits - #8045. Thanks @say25!",
+      "[Fixed] Tiny images in two-up image rendering - #8103",
+      "[Fixed] Manual conflict resolution for binary files always chose same version - #8059",
```

Present tense elsewhere so made this one present as well for consistency
```suggestion
      "[Fixed] Manual conflict resolution for binary files always choose same version - #8059",
```